tests/worker/test_advertise_ip.py

```
@@ -0,0 +1,68 @@
+
+    def json(self):
+        return {"status": "registered"}
+
+    def raise_for_status(self):
+        return None
+
+
+class _CaptureClient:
+    """Minimal async httpx.AsyncClient stand-in that records the POST body."""
+
+    captured = {}
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)